app/js/main.js

```diff
@@ -167,20 +203,26 @@ stitchButton.addEventListener(CLICK_EVENT, (e) => {
     };
 
     [...imagesList.children].forEach(tr => {
-        const img = new Image();
-        images.push(img);
-        img.src = tr.dataset[DATA_FILE];
-        img.addEventListener(LOAD_EVENT, () => {
-            minX = Math.min(minX, img.width);
-            maxX = Math.max(maxX, img.width);
-            minY = Math.min(minY, img.height);
-            maxY = Math.max(maxY, img.height);
-            sumX += img.width;
-            sumY += img.height;
-
-            if (++loaded === imagesList.children.length) {
-                stitchImages();
-            }
+    fetch(tr.dataset[DATA_FILE])
+        .then(response => response.blob())
+        .then(blob => createImageBitmap(blob))
+        .then(bitmap => {
+        images.push(bitmap);
```

Comment:
Rename variable from `images` to `bitmaps´, because content changed.

Reply:
Fixed in [2ada9af](https://github.com/markuszeller/image-stitcher/pull/14/commits/2ada9af46a5c2e1338c43f68eda11ca13d234ea5)